Lib/ctypes/test/test_loading.py

```diff
@@ -24,10 +24,20 @@ class LoaderTest(unittest.TestCase):
     unknowndll = "xxrandomnamexx"
 
     def test_load(self):
-        if libc_name is None:
-            self.skipTest('could not find libc')
-        CDLL(libc_name)
-        CDLL(os.path.basename(libc_name))
+        if libc_name is not None:
+            test_lib = libc_name
+        else:
+            if os.name == "nt":
+                import _ctypes_test
+                test_lib = _ctypes_test.__file__
+            else:
+                self.skipTest('could not find libc')
```

Comment:
```suggestion
                self.skipTest('could not find library to load')
```
Since we're no longer loading just libc.